data/npc/lib/npcsystem/modules.lua

```diff
@@ -945,7 +944,7 @@ if Modules == nil then
 		end
 
 		local subType = shopItem.subType or 1
-		local a, b = doNpcSellItem(cid, itemid, amount, subType, ignoreCap, inBackpacks, backpack)
+		local a, b = doNpcSellItem(cid, itemid, amount, subType, ignoreCap, inBackpacks, ITEM_SHOPPING_BAG)
```

Comment:
Is `ITEM_SHOPPING_BAG` declared somewhere in Lua where I can't find it? Or are you missing a `registerEnum(ITEM_SHOPPING_BAG)` call in `luascript.cpp`?

Reply:
thanks